regulations/static/regulations/js/source/views/comment/comment-view.js

```diff
@@ -77,13 +82,32 @@ var CommentView = Backbone.View.extend({
   },
 
   target: function(options) {
+    if (options.section) {
+      var parsed = helpers.parsePreambleId(options.section);
+      var href = window.APP_PREFIX + parsed.path.join('/') + '#' + parsed.hash;
+      this.$header.html('<a href="' + href + '">' + options.label + '</a>');
+    }
     this.setSection(options.section, options.tocId, options.label);
     this.$context.empty();
     if (options.$parent) {
       this.$context.append(options.$parent);
     }
   },
 
+  openComment: function(e) {
+    e.preventDefault();
+    var options = {
+      section: this.model.get('id'),
+      tocId: this.model.get('tocId'),
+      label: this.model.get('label')
+    };
+    // TODO: Push this logic into `PreambleView`
+    var type = options.section.split('-')[1];
+    DrawerEvents.trigger('section:open', options.tocId);
+    DrawerEvents.trigger('pane:change', type === 'preamble' ? 'table-of-contents' : 'table-of-contents-secondary');
+    MainEvents.trigger('section:open', options.section, options, 'preamble-section');
```

Comment:
+1 to combining this with PreambleView; we basically want this to trigger identical events as if the user had clicked the "Read" tab.